system_tests/resources/start-ec2-worker-vm-blueprint.yaml

```diff
@@ -60,12 +64,23 @@ node_templates:
       aws_config: *AWS_CONFIG
 
   server:
-    type: cloudify.aws.nodes.Instance
+    type: cloudify.aws.nodes.WindowsInstance
     properties:
       install_agent: false
       image_id: { get_input: image_id }
       instance_type: { get_input: instance_type }
       aws_config: *AWS_CONFIG
+      parameters:
+        user_data: |
+          <powershell>
+          &winrm quickconfig -q
```

Comment:
Well, probably we could loose some (I just copied it from this other blueprint - better safe than sorry). However, it appears that we need at least 2 of them (the 5985 line obviously and maybe the basic=true or the allowunencrypted), since I've tried it with only the 5985 and it says there's a problem authenticating with simple password on a different domain in windows - I've already talked with nirbir about it.